CMakeLists.txt

```diff
@@ -921,6 +922,7 @@ if(WITH_TESTS)
         table/cleanable_test.cc
         table/cuckoo_table_builder_test.cc
         table/cuckoo_table_reader_test.cc
+        table/data_block_hash_index.cc
```

Comment:
This should be data_block_has_index_test.cc ... and that's why the windows build failed.